### PR TITLE
[TR] Translation of index.md

### PR DESCRIPTION
**Pull request by mekici**

### CODE REVIEW

**Comment by Atcold** [comment]
We already have translated `index.md`. What is this PR about?

**Comment by Atcold** [comment]
Oh… I think you need to rebase on top of `master`.

File: docs/tr/index.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 ---
 layout: default
+<<<<<<< HEAD
```
Comment (Atcold) [inline docs/tr/index.md:3]: Garbage to take care of.

File: docs/tr/index.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
+title: pytorch ile derin öğrenme
+=======
```
Comment (Atcold) [inline docs/tr/index.md:5]: More garbage.

File: docs/tr/index.md [hunk 3]
```diff
@@ -1,6 +1,10 @@
...
 title: DERIN ÖĞRENME
+>>>>>>> b75959425fbd710c8e11004f6c91905343f98c78
```
Comment (Atcold) [inline docs/tr/index.md:7]: More here.

File: docs/tr/index.md [hunk 4]
```diff
@@ -1,6 +1,10 @@
 ---
 layout: default
+<<<<<<< HEAD
+title: pytorch ile derin öğrenme
+=======
 title: DERIN ÖĞRENME
```
Comment (Atcold) [inline docs/tr/index.md:6]: This is the correct title.

File: docs/tr/index.md [hunk 5]
```diff
@@ -236,6 +240,82 @@ Bu kurs,gözetimli ve gözetimsiz derin öğrenme, gömme yöntemleri, ölçev 
 
 
 
+<<<<<<< HEAD
```
Comment (Atcold) [inline docs/tr/index.md:243]: Garbage.

File: docs/tr/index.md [hunk 6]
```diff
@@ -236,6 +240,82 @@ Bu kurs,gözetimli ve gözetimsiz derin öğrenme, gömme yöntemleri, ölçev 
...
+    </tr>
+    <tr>
+      <td style="text-align: left"> Grader </td>
+      <td style="text-align: center"><img src="https://st3.depositphotos.com/13159112/17145/v/450/depositphotos_171453724-stock-illustration-default-avatar-profile-icon-grey.jpg" width="100" height="100" /></td>
+      <td style="text-align: left">Bixing Yan <br />by783@nyu.edu</td>
+      <td style="text-align: left"> </td>
+    </tr>
+  </tbody>
+</table>
+=======
```
Comment (Atcold) [inline docs/tr/index.md:318]: Garbage.

File: docs/tr/index.md [hunk 7]
```diff
@@ -251,3 +331,4 @@ Bu kurs,gözetimli ve gözetimsiz derin öğrenme, gömme yöntemleri, ölçev 
 |Grader|<img src="https://st3.depositphotos.com/13159112/17145/v/450/depositphotos_171453724-stock-illustration-default-avatar-profile-icon-grey.jpg" width="100" height="100">|Pedro Vidal<br>pmh314@nyu.edu|
 |Grader|<img src="https://st3.depositphotos.com/13159112/17145/v/450/depositphotos_171453724-stock-illustration-default-avatar-profile-icon-grey.jpg" width="100" height="100">|Bixing Yan <br>by783@nyu.edu|
 
+>>>>>>> b75959425fbd710c8e11004f6c91905343f98c78
```
Comment (Atcold) [inline docs/tr/index.md:334]: Garbage.

**Comment by Atcold** [comment]
`index.md` is still polluted (from a failed merge conflict) and there is still an edit to the website format.

image

File: docs/tr/index.md [hunk 8]
```diff
@@ -1,10 +1,10 @@
 ---
 layout: default
-title: DERIN ÖĞRENME
+title: Pytorch ile derin öğrenme
```
Comment (Atcold) [inline docs/tr/index.md:3]: Revert.

File: docs/tr/index.md [hunk 9]
```diff
@@ -235,19 +236,78 @@ Bu kurs,gözetimli ve gözetimsiz derin öğrenme, gömme yöntemleri, ölçev 
 
 
 
+## Ekip
```
Comment (Atcold) [inline docs/tr/index.md:239]: I think the issue is with the table above. There is something that breaks all the table down.
Have you tried running the website locally?

**Comment by mekici** [comment]
Hello Alfredo,

I am so sorry about the conflicted merge. I changed the table before and it
looks fine even in your docs folder. Please take a look:
https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/tr/index.md
But
somehow on this link https://atcold.github.io/pytorch-Deep-Learning/tr/ an
older version of the file is running. The one in your docs folder has a
table for people (please see the page source) but the one in use not. This
is why the people and the lecture table are mixed up. I reverted the commit
changing the layout.

**Comment by Atcold** [comment]
> Please take a look: https://github.com/Atcold/pytorch-Deep-Learning/blob/master/docs/tr/index.md

This *is* the file that *jekyll* is trying to convert to `HTML`, but there are errors, as you can see from the homepage. It is *not* using an older version. It is using the same version. *Jekyll* is reading the `MD` file but then fails to generate the conversion because of some issues with the table formatting. The fact that GitHub is able to visualise it correctly doesn't mean the file does not have errors.

Can you plz try to render the website locally and see where is the bug?

**Comment by Atcold** [comment]
Any update here?

**Comment by mekici** [comment]
Dear Alfredo,

I removed the fork and tried to push it again. I hope it works now.  I
tested the file locally before pushing.  Please check it and let me know if
there is any problem. Take care!

[image: Screen Shot 2020-07-17 at 17.59.38.png]

Regards,
_________________________________________________



Dr. Murat EKİCİ, Ph.D.
Usak University
Faculty of Education
Department Of Computer Education and Instructional Technology, Office #329
USAK/TURKEY

**Comment by Atcold** [comment]
Did you create a *new* PR? Is that what you meant?
Here I still see the change in the submodule.

**Comment by mekici** [comment]
Sorry for the belated response I was on vacation.  Yes, I created a new
pull request (#420 ) and everything seems to be working and aligned.

Regards,
_________________________________________________



Dr. Murat EKİCİ, Ph.D.
Usak University
Faculty of Education
Department Of Computer Education and Instructional Technology, Office #329
USAK/TURKEY

**Comment by Atcold** [comment]
If you are okay, please close the PR.

**Pull request closed without merging**